skills/cairo-auditor/tests/validate_bundle_generation.py

```
@@ -0,0 +1,140 @@
+                f"bundle {bundle} does not contain attack-vectors-{i} partition "
+                f"(missing first line: {partition_first_line!r})",
+            )
+    return True, "all 4 bundles exist, are non-empty, and contain required sections"
+
+
+def main() -> int:
+    if not FIXTURE.exists():
+        print(f"missing fixture: {FIXTURE}", file=sys.stderr)
+        return 1
+    if not (REFS / "judging.md").is_file():
+        print(f"missing references/judging.md", file=sys.stderr)
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)